### PR TITLE
Homework week2

### PR DESCRIPTION
**Pull request by benna100**

### CODE REVIEW

File: JavaScript1/Week2/homework.md [hunk 1]
```diff
@@ -6,200 +6,157 @@
 - Go through the review of [last week](../Week1/review.md) (Work in progress, update this week :wrench:)
 - Go through the review of [this week](../Week2/review.md) (work in progress, update this week :nut_and_bolt:)
```
Comment (abedsujan) [inline JavaScript1/Week2/homework.md:8]: Looks good and very organized by step by step building block with hints. 
Excellent work! - good to be merge.

**Review by abedsujan — APPROVED** [review]

File: JavaScript1/Week2/homework.md [hunk 2]
```diff
@@ -6,200 +6,157 @@
...
 
-## Step 2: JavaScript
-> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference
 
-1. Create a function that takes 3 arguments and returns the sum of the three arguments.
+## Step 2: Javascript warmup part one
+Just like last time, lets warmup our brain!
+Go back to FreeCodeCamp, start where you left of and finish the rest of the Basic JavaScript challenges. We know this seems like a lot, but the tasks are not so big, so hang in there!
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: Not sure if typos are even worth mentioning? Should say "..where you left off.."

File: JavaScript1/Week2/homework.md [hunk 3]
```diff
@@ -6,200 +6,157 @@
...
-var z = y;
-```
+Hint: use remainder operator
 
-What do you think will happen with `x == y`, `x === y` and `z == y` and `z == x`? Prove it!
-    
-> Don't cheat! Seriously - try it first.
-    
+## Step 3: Smart-ease - Goes Global!
+Following the massive success of Smart-ease's first couple of products things have been quite hectic at Smart-ease's shared office space. The phone is constantly ringing from journalists wanting to interview the genius behind the success and the greatest developers want to work for you. Wired wrote an article with the headline "Smart-ease as good as Smarties"
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: Could add a link to Wired.com, just so people understand it as a well-known technology news outlet.

File: JavaScript1/Week2/homework.md [hunk 4]
```diff
@@ -6,200 +6,157 @@
...
+As always Smart-ease starts by helping people fix their problems. So lets venture out into the real world and see what we find:
 
+### NOnoN0nOYes (Note taking app)
+> You are sitting at a meeting. A person is presenting some interesting thought that you want to write down. You check your bag, but realise that you forgot to bring your notepad. Hmm you check the internet for an online solution. But you need to signup for them all, too bad... Maybe this could be a worthwhile problem so solve. Lets create a simple and easy to use notepad 📝
 
-16. Take a look at the following code: 
+#### Save a note
+The first thing we will create is the functionality to save a note:
+Create a variable called notes and assign it to an empty array.
+Create a function called addNote. The addNote has two parameters: note and noteId. The note parameter is a string. and the noteId is a number.
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: should the variables, functions etc be highlighted as code? Just a thought for even easier reading.

File: JavaScript1/Week2/homework.md [hunk 5]
```diff
@@ -6,200 +6,157 @@
...
+Optional
+* Lets improve the addActivity, so that we dont need to specify the date, but the function automatically figures out what the date is. Check out this link: https://stackoverflow.com/a/34015511
+* Improve the ShowStatus function by only showing the number of actitivies for that specific day.
+* Create a function for calculating the activity a user has spent the most time on.
+---
 
 
-**3 (Challenge - only do this one once you have finished 1 and 2). Please work closely with your pair we identified in class.**
+## Step 4: Rover the Robot (Optional)
+Go and try out this cool game: [roverjs.com](http://roverjs.com), written by one of the HYF teachers from Amsterdam, Joost Lubach. There are different levels. Solve the chapter up until and with the variables chapter. There might be a small price for completing that!
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: "The chapter up until when?" I can already hear Omid yelling 😆

File: JavaScript1/Week2/homework.md [hunk 6]
```diff
@@ -6,200 +6,157 @@
...
 
-I would strongly recommend play the game couple of times and adjust the step below if 
-necessary. See also the flow chart we gave you.
+## Step 5: Hand in Homework:
+Steps:
+* Use the hyf-homework repo and add your homework files in the javascript/javascript1/week2 folder
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: Maybe again highlight the repo?

File: JavaScript1/Week2/homework.md [hunk 7]
```diff
@@ -6,200 +6,157 @@
...
-3. Create an object for ``game`` with the following variables: array of four players and four arrays of cards.
-4. Each array belongs to one player.
-5. Create a function to distribute the cards to the four arrays.
-6. Take the next player in the array. 
-7. Find cards with highest frequency.
-8. Go to the next player and see if the player has the card you want
-9. If it does continue and go to 7. If not go to 6.
-10. If at any point there is one happy family in one of the array change the winner key to "true"
-declare winner.
+And thats it your are done 🎉
```
Comment (chklueter) [inline JavaScript1/Week2/homework.md:163]: That was AWESOME!! Another amazing homework. WHO WOULDN'T WANT TO DO THIS?!?!! I wonder if people will understand the Smarties pun 🤔 https://www.quiltsmart.com/media/ecom/prodsm/WEB%20bb%20cover.jpg

File: JavaScript1/Week2/homework.md (hunk 5 above)
Comment (chklueter) [inline JavaScript1/Week2/homework.md]: maybe we could link to Joost's github page so people can check his stuff out?

**Review by chklueter — APPROVED** [review]

File: JavaScript1/Week2/homework.md (hunk 2 above)
Comment (benna100) [reply JavaScript1/Week2/homework.md]: Yes, we should do typos aswell.

File: JavaScript1/Week2/homework.md (hunk 7 above)
Comment (benna100) [reply JavaScript1/Week2/homework.md:163]: Ah, that's funny with the actual smarties cover :) 
Maybe some will, if they dont get it, the name still sounds kind of techy, so it does not matter i guess.

File: JavaScript1/Week2/homework.md (hunk 5 above)
Comment (benna100) [reply JavaScript1/Week2/homework.md]: Yeah, good idea :)

**Review by chklueter — APPROVED** [review]

**Pull request merged**